services/osparc-gateway-server/src/osparc_gateway_server/backend/osparc.py

```diff
@@ -85,26 +95,29 @@ async def do_start_cluster(
         modifications = get_osparc_scheduler_cmd_modifications(scheduler_service_name)
         for key, value in modifications.items():
             scheduler_cmd = modify_cmd_argument(scheduler_cmd, key, value)
-
+        # start the scheduler
+        # asyncio.create_task(_background_task(self, cluster))
         async for dask_scheduler_start_result in start_service(
-            self.docker_client,
-            self.settings,
-            self.log,
+            docker_client=self.docker_client,
+            settings=self.settings,
+            logger=self.log,
             service_name=scheduler_service_name,
             base_env=scheduler_env,
             cluster_secrets=[
                 c for c in self.cluster_secrets if c.cluster.name == cluster.name
             ],
             cmd=scheduler_cmd,
-            labels={"cluster_id": f"{cluster.id}"},
+            labels={"cluster_id": f"{cluster.id}", "type": "scheduler"},
             gateway_api_url=self.api_url,
         ):
             yield dask_scheduler_start_result
 
     async def do_stop_cluster(self, cluster: Cluster):
+        self.log.debug("<-- stopping cluster %s", f"{cluster=}")
```

Comment:
MINOR: the word ``cluster`` is repeated twice in the log message .Same issue in logs below with cluster, worker ...